lmctl/project/handlers/etsi_ns/etsi_ns_content.py

```diff
@@ -0,0 +1,85 @@
+import os
+import lmctl.project.handlers.assembly as assembly_api
+from lmctl.project.validation import ValidationResult, ValidationViolation
+import lmctl.utils.descriptors as descriptors
+import lmctl.drivers.lm.base as lm_drivers
+
+
+class EtsiNsPkgContentTree(assembly_api.AssemblyPkgContentTree):
+    def __int__(self, root_path=None):
+        super().__init__(root_path)
+
+    MANIFEST_FILE = 'MRF.mf'
+    FILES_DIRECTORY = 'Files'
+    DEFINITIONS_DIR = 'Definitions'
+    
+    @property
+    def manifest_file_path(self):
+        mf_path = self.resolve_relative_path(EtsiNsPkgContentTree.MANIFEST_FILE)
+        if os.path.exists(mf_path):
+            return mf_path
+
+    @property
+    def files_dir_path(self):
+        files_path = self.resolve_relative_path(EtsiNsPkgContentTree.FILES_DIRECTORY)
+        if os.path.exists(files_path):
+            return files_path
+
+    @property
+    def definitions_dir_path(self):
+        definitions_dir = self.resolve_relative_path(EtsiNsPkgContentTree.DEFINITIONS_DIR)
+        if os.path.exists(definitions_dir):
+            return definitions_dir
+
+    @property
+    def test_config_dir_path(self):
+        return self.resolve_relative_path(assembly_api.AssemblyPkgContentTree.BEHAVIOUR_TESTS_DIR_NAME)            
+
+    @property
+    def descriptor_definitions_file_path(self):
+        return self.resolve_relative_path(EtsiNsPkgContentTree.DEFINITIONS_DIR, EtsiNsPkgContentTree.DESCRIPTOR_FILE_YML)
+    
+    def gen_full_csar_package_file_path(self, resource_name, resource_version):
+        if os.path.exists(self.relative_path('_lmctl/build/{0}-{1}.csar'.format(resource_name, resource_version))):
```

Comment:
Can see why this is done (and works) but if we build a package and share it to someone else who uses `lmctl pkg push` then I don't think this will work. 
I guess the problem here is lmctl extracts the package and expects to push the contents inside when actually when using ETSI we just want to push the whole thing. We could wrap the ETSI package inside an lmctl package (so lmctl can extract the outer package to reveal the ETSI package) but I can imagine this isn't the desired behaviour, as we'd prefer the package produced by lmctl to be ETSI compliant out-of-the-box and without requiring lmctl.

This may need a bigger adjustment to the push process unfortunately. We can discuss this when available.